crates/primitives/src/transaction/tx_type.rs

```diff
@@ -27,6 +27,8 @@ pub enum TxType {
     EIP2930 = 1_isize,
     /// Transaction with Priority fee
     EIP1559 = 2_isize,
+    /// Shard Blob Transactions - EIP-4844
+    EIP4844 = 3_isize,
```

Comment:
I think we can do this already

because does not conflict with the existing database

any objections @joshieDo ?